internal/fs/fs.go

```diff
@@ -280,6 +287,22 @@ func copyFile(src, dst string) (err error) {
 	return
 }
 
+// copySymlink will resolve the src symlink and create a new symlink in dst.
+// If src is a relative symlink, dst will also be a relative symlink.
+func copySymlink(src, dst string) error {
+	resolved, err := os.Readlink(src)
+	if err != nil {
+		return errors.Wrap(err, "failed to resolve symlink")
+	}
+
+	err = os.Symlink(resolved, dst)
+	if err != nil {
```

Comment:
you can write this 

return errors.Wrapf(err, "...") 

if err == nil, errors.Wrapf returns nil.

This might be too magical.